config/quota-management-list-configuration.yaml

```diff
@@ -5,25 +5,64 @@
 # In the future it may support service accounts outside RH.
 # The structure of registered service accounts is:
 #       - username: is the account of the user. The username must be unique
-#       - max_allowed_instances: is the maximum number of instances this user can create.
+#       - max_allowed_instances: is the maximum number of instances of a given instance_type/billing_model pair this user can create.
+#         This value is used if `allowed` field (inside `billing_model`) is 0 or omitted.
 #         Defaults to the global value of `max-allowed-instances` which has different values for distinct environments.
+#       - granted_quota: the configuration of the quota granted to the user. The configuration is defined as follows:
+#         - instance_type_id: the id of the instance type for which we are configuring the quota
+#         - billing_models: the list of billing models that we are going to configure.
+#           The set of accepted billing models, depends on the KafkaBillingModel configuration.
```

Comment:
There's an open comment by @miguelsorianod around the naming of this field (see https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1368#discussion_r1022703770).

Since this is talking about the KafkaBillingModel it might be better to just rename this as `kafka_billing_model` , wdyt?